src/Range.tsx

```diff
@@ -60,7 +67,7 @@ class Range extends React.Component<IProps> {
 
   componentDidMount() {
     const { values, min, step } = this.props;
-    window.addEventListener('resize', this.schdOnWindowResize);
+    window.addEventListener('resize', this.schdOnResize);
```

Comment:
Are we now triggering both `resize` event and `ResizeObserver`? This should not be added if ResizeObserver is supported?

Reply:
I moved the resize event listeners to the `observe` and `unobserve` functions of the fallback object.